cycle1/.gitignore

```diff
@@ -0,0 +1 @@
+index.html
```

Comment:
I removed it on purpose to not ignore the one `public/index.html`. Just delete your local one and run `elm-live` as described in the README.